src/entity/Living.php

```diff
@@ -440,6 +441,10 @@ public function applyDamageModifiers(EntityDamageEvent $source) : void{
 		$source->setModifier(-$source->getFinalDamage() * min(ceil(min($totalEpf, 25) * (mt_rand(50, 100) / 100)), 20) * 0.04, EntityDamageEvent::MODIFIER_ARMOR_ENCHANTMENTS);
 
 		$source->setModifier(-min($this->getAbsorption(), $source->getFinalDamage()), EntityDamageEvent::MODIFIER_ABSORPTION);
+
+		if($cause === EntityDamageEvent::CAUSE_FALLING_BLOCK && $this->armorInventory->getHelmet() instanceof Armor){
+			$source->setModifier(-min($source->getFinalDamage() / 4, 10), EntityDamageEvent::MODIFIER_HARD_HELMET);
```

Comment:
My research did not show any minimum of 10.

Reply:
> My research did not show any minimum of 10.

done.